<!DOCTYPE html>
<!-- DO NOT EDIT! This test has been generated by /html/canvas/tools/gentest.py. -->
<title>Canvas test: toDataURL.arguments.2</title><title>Canvas test: toDataURL.arguments.2</title>
<script src="/resources/testharness.js"></script>
<script src="/resources/testharnessreport.js"></script>
<script src="/html/canvas/resources/canvas-tests.css">
<title><h1><h1><h1><h1>toDataURL.arguments.2</h1></h1></h1></h1></title><title><h1><h1><h1><h1>toDataURL.arguments.2</h1></h1></h1></h1></title><title><h1><h1><h1><h1>toDataURL.arguments.2</h1></h1></h1></h1></title><title><h1><h1><h1><h1>toDataURL.arguments.2</h1></h1></h1></h1></title><title><h1><h1><h1><h1>toDataURL.arguments.2</h1></h1></h1></h1></title><title><h1><h1><h1><h1>toDataURL.arguments.2</h1></h1></h1></h1></title>

<body class="show_output">
<p class="desc">toDataURL ignores extra arguments</p>


<p class="output">Actual output:</p>
<canvas id="c" class="output" width="100" height="50"><p class="fallback">FAIL (fallback content)</p></canvas>

<ul id="d"></ul>
<script>
var t = async_test("toDataURL('image/png', 'another argument that should not raise an exception', 'and another');
assert_regexp_match(data, /^data:image\/png[;,]/);


});
</script>

